### ISSUE TITLE
Open A11y Pilot: Upstream issues

### ISSUE THREAD

**Issue by nschonni** [mgifford/a11y-contracting#3]
> Should accessibility issues be found "up-stream" in libaries used by the project, efforts should be made to ensure that:

- This might need some time boxing. EX: 3 months after the code gets handed over or on the last day might not be that helpful
- Also brings up what happens to the forks for those patches at the end. Can they just close them and delete their forks, or do they also get transfered

**Comment by mgifford** [comment]
I could fix the first point with "- patches supplied by the community are tested, if they resolve the pattern, then this should be added to the issue queue before the project is finalized. "

I'm not sure how to deal with the forks though. Any ideas?
